CWE-94-Python_darwin/andy897221_Proof-of-Play-Flow-Demo/PoP.py

```
@@ -0,0 +1,107 @@
+                # myIP: provide worldwide IP if the blockchain deployment is global, local IP otherwise
+                # the above two argument is disabled for simplify features
+                blockchain = \
+                    _blockchain.main(self.nodeID, self.blockchain_port, self.blockchain_bootstrap_ip, self.blockchainLoc,
+                                     self.keyLoc, saveState, self.ratingFunc, auto_broadcast)
+                blockchain.run_app(func)
+            return func_wrapper
+        return user_func
+
+    def game_conn_to(self, bootstrap_addr):
+        # bootstrap_addr: the node to connect to retrieve other nodes, None if you are the bootstrap
+        return requests.post(f"http://127.0.0.1:{self.api_port}/conn",
```

Ruff (0.13.2)

Probable use of `requests` call without timeout

(S113)